### PR TITLE
Fixed documentation URL for PrestaShop 9.0.x

### LINKED ISSUE #37119: On develop branch aka 9.0.0, the documentation targets the v8
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

On develop branch aka 9.0.0, the documentation targets the v8

(Just checked on this page BO > International > Localization > Currencies)

### Expected behavior

On develop branch aka 9.0.0, the documentation targets the v9

### Steps to reproduce

image

### PrestaShop version(s) where the bug happened

9.0.x

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

@PrestaShopCorp

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x 
| Description?      | Fixed documentation URL for PrestaShop 9.0.x
| Type?             | bug fix 
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | N/A
| UI Tests          | N/A
| Fixed issue or discussion?     | Fixes #37119
| Related PRs       | N/A
| Sponsor company   | PrestaShop SA

### CODE REVIEW

File: admin-dev/themes/new-theme/buildLock
Comment (boherm) [inline admin-dev/themes/new-theme/buildLock:1]: I don't thing that we need this file 🤔

**Review by boherm — CHANGES_REQUESTED** [review]
LGTM, but just one comment about `buildLock` file in your PR ;)

File: admin-dev/themes/new-theme/buildLock
Comment (nicosomb) [reply admin-dev/themes/new-theme/buildLock:1]: Don't know why this file was here.

**Review by jolelievre — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by tleon — APPROVED** [review]

**Comment by nicosomb** [comment]
No QA needed.

**Pull request merged**